src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  switch (true) {
+    case days < 3:
+      return days * 40;
```

Comment:
Replace the literal '40' with a constant to give it a meaningful name, like 'DAILY_RATE'.